.github/workflows/rosa-cluster-auto-provision-on-schedule.yml

```diff
@@ -52,14 +52,13 @@ jobs:
 
   keycloak-deploy-active-active:
     needs: keycloak-undeploy-active-active-with-external-infinispan
-    name: ROSA Scheduled Create Active/Active cluster with Persistent Sessions
+    name: ROSA Scheduled Create Active/Active cluster with External Infinispan and Persistent Sessions
     if: github.event_name != 'schedule' || github.repository == 'keycloak/keycloak-benchmark'
     uses: ./.github/workflows/rosa-multi-az-cluster-create.yml
     with:
       clusterPrefix: gh-keycloak # ${{ env.CLUSTER_PREFIX }} -- unfortunately 'env.' doesn't work here
-      enablePersistentSessionsFeature: true
```

Comment:
How do we enable persistent sessions?

Reply:
@ryanemerson - thank you for the explanation. Assuming this is now resolve, I'll go a head and merge this. 

@ryanemerson / @mhajas - it would be good to figure out how this would be communicated (both within the team, and then to users who want to try out this experimental feature).

Reply:
Persistent sessions can still be enabled using `persistent-user-session` feature. Multi-site is just another option how to enable it.